cassiopeia/data/utilities.py

```diff
@@ -108,3 +111,47 @@ def _to_newick_str(g, node):
 
     root = [node for node in tree if tree.in_degree(node) == 0][0]
     return _to_newick_str(tree, root) + ";"
+
+def compute_dissimilarity_map(
+    cm: np.array,
+    C: int,
+    dissimilarity_function: Callable,
+    weights: Optional[Dict[int, Dict[int, float]]] = None,
+    missing_indicator: int = -1,
```

Comment:
This is called `missing_state_indicator` in the `CassiopeiaTree`, I think we should use consistent naming access the whole codebase.

Reply:
Good point, I'll go through and change these.